src/libs/TransactionUtils.ts

```diff
@@ -0,0 +1,358 @@
+import Onyx, {OnyxCollection} from 'react-native-onyx';
+import {format, parseISO, isValid} from 'date-fns';
+import CONST from '../CONST';
+import ONYXKEYS from '../ONYXKEYS';
+import DateUtils from './DateUtils';
+import * as NumberUtils from './NumberUtils';
+import {RecentWaypoints, ReportAction, Transaction} from '../types/onyx';
+import {Receipt, Comment} from '../types/onyx/Transaction';
+import {OriginalMessageIOU} from '../types/onyx/OriginalMessage';
+
+type TransactionChanges = Partial<Transaction> & {comment?: string};
+
+type OriginalMessage = OriginalMessageIOU['originalMessage'];
+
+type Waypoints = Record<string, RecentWaypoints>;
+
+let allTransactions: OnyxCollection<Transaction> = {};
+
+Onyx.connect({
+    key: ONYXKEYS.COLLECTION.TRANSACTION,
+    waitForCollectionCallback: true,
+    callback: (val) => {
```

Comment:
let's change it to `value`

Reply:
@kubabutkiewicz In most of the similar cases it's `val` so I would leave it as is to keep it consistent.
![image](https://github.com/Expensify/App/assets/23176449/ecdc3e5f-02de-486c-9329-ba6b6cbf7b1b)

Reply:
Yes, however you can see that all of these files are JS.

Reply:
During TS migration it would be great to migrate to normal names as well.

Reply:
Any variable that you can see looks strange like `val`, `t`, `a`, `b`, `c`, `v`, etc, we should rename.

Reply:
Got it, thank you